src/components/pages/test-pages/mui-alert-page/MUIAlertPage.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { Box } from '@mui/material';
+
+import Alert from '@/components/common/ui/alert-mui';
+import {
+  AlertType,
+  AlertVariantType,
+} from '@/components/common/ui/alert-mui/Alert';
+
+import * as styles from './MUIAlertPage.styles';
+
+const colors: AlertType[] = ['info', 'success', 'warning', 'error'];
+const variants: AlertVariantType[] = [
+  'filled',
+  'outlined',
+  'darker',
+  'border-top',
+  'border-left',
+];
+
+const MUIAlertPage = () => {
+  return (
+    <Box sx={styles.wrapper}>
+      {colors.map(color => (
+        <>
```

Comment:
remove `<>`

Reply:
I need to wrap inside map in react fragment to make this one react node (otherwise - error)